docs/home/howto/CLUSTER.md

```diff
@@ -291,6 +291,21 @@ To setup the cluster do the following steps from the provisioning machine:
     epicli apply -f newcluster.yml
     ```
 
+## How to create an Epiphany cluster into an existing network (Azure only)
+
+An existing network can be specified in the cluster's YAML file as follows:
+
+```yaml
+network:
+      existing_rg: resource-group-name
+      existing_vnet: vnet-name
+      existing_subnet: subnet-name
+```
+
+If these values are specific, the `use_network_security_groups` will be ignored (i.e. effectively always be false).
```

Comment:
@ks4225 sorry for late response. Did you mean `specified` instead of `specific`? I'm trying to understand intent of that change first.

Reply:
Yes, sorry, typo there. It should be "specified".

Reply:
so that should be fixed.